### ISSUE TITLE
In cloud functions, request.user is undefined

### ISSUE THREAD

**Issue by natanrolnik** [ParsePlatform/parse-server#90]
In a few of my Cloud functions, I have the following code, to assert that the request is being made by an authenticated user:

```
var currentUser = request.user;

  if (!currentUser) {
    response.error("You must be authenticated to call queryClinic");
    return;
  }
```
When using parse-server, currentUser doesn't exists. Is this a temporary limitation?

**Comment by gfosco** [comment]
Yeah, this should be a quick fix.  `functions.js` has a TODO for it. :)

**Comment by natanrolnik** [comment]
Awesome! @gfosco thanks for the hard work on making parse-server accessible for us, future-Parse-orphans!

**Comment by gfosco** [comment]
Will be addressed in #97.
